### DISCUSSION TITLE
[RESOLVIDO] Payoneer para recebimentos em conta PJ

### DISCUSSION THREAD

**Discussion by GiovanniZambiasi** [DyegoCosta/trabalhando-remoto#177] [General]
Olá!

Utilizo o Payoneer para receber pagamentos em dólar do meu cliente (do Reino Unido). Tenho uma conta jurídica no Banco Inter, e vinha conseguindo receber no primeiro semestre de 2019. Depois de passar alguns meses sem usar o serviço, tentei fazer um saque para minha conta, como de costume. No entanto, após 5 dias úteis recebo um email dizendo que o requerimento falhou. Estou tentando resolver o problema, mas temo que não seja possível. Se for o caso, preciso de recomendações de bancos para abertura de conta PJ. Acredito que esses bancos digitais tenham maiores dificuldades com câmbio, visto que o próprio banco Inter para fazer remessas internacionais te dá uma dor de cabeça atrás da outra. Alguém teria recomendação de alguma conta PJ que funcione com Payoneer? Se não, alguma conta PJ que me permita receber em dólar sem cobrar nenhuma taxa muito alta? 

Desde já agradeço!
Giovanni

**EDIT [SOLUÇÃO]**: 
https://github.com/DyegoCosta/trabalhando-remoto/issues/83#issuecomment-560533831

**Comment by clmadrazo** [comment]
opa @GiovanniZambiasi eu recebo pelo payoneer e transfiro para minha conta PJ no banco Inter desde maio do ano passado e até agora nunca tem dado problemas. O problema de vc é com o Payoneer ou com o banco Inter? realmente não tenho encontrado uma opção melhor que o payoneer para transferir para contas PJ

**Comment by GiovanniZambiasi** [comment]
> opa @GiovanniZambiasi eu recebo pelo payoneer e transfiro para minha conta PJ no banco Inter desde maio do ano passado e até agora nunca tem dado problemas. O problema de vc é com o Payoneer ou com o banco Inter?

Pois é cara, eu não faço ideia. Recebi vários pagamentos Payoneer>Banco Inter, mas nesse mês todos os meus requests tem falhado. Imaginei que fosse coisa do banco, então liguei para o SAC, que me disse que não tinham as informações, e me redirecionaram ao SAC do Payoneer. O Payoneer para passar informações é uma novela também, demoram muito. Agora estou na fila do SAC mais uma vez para tentar descobrir.
Tentou fazer alguma transferencia nesse mês já? Deu certo?

**Comment by clmadrazo** [comment]
Sim, transferi sem problemas no início do mês. Qualquer questão com o payoneer eu sempre resolvo pelo chat mesmo

**Comment by GiovanniZambiasi** [comment]
Bom saber que ainda tenho esperanças de seguir usando minha conta Inter. Gosto bastante das opções deles. Vou tentar resolver com o Payoneer. Obrigado pela resposta, @clmadrazo!

**Comment by igorbp** [comment]
Oi, @GiovanniZambiasi. Eu também faço o mesmo processo. Payoneer > Inter PJ e tive problemas 2 meses atrás. Foi com um banco intermediário que trabalha com a Payoneer. Banco Rendimento. Tive que falar com a Payoneer para o banco finalmente entrar em contato comigo e me explicar que eles precisavam de alguns documentos a mais. Além de exigirem agora a invoice todos os meses. Foi ruim porque o pagamento ficou preso lá até que eu fosse procurá-los . Ninguém pensou em me mandar um e-mail explicando a situação . Mas enfim, entra em contato com a Payoneer que eles devem pelo menos contatar o banco para falarem com você.

**Comment by GiovanniZambiasi** [comment]
Obrigado pela info, @igorbp! Acabei de contactar o SAC da Payoneer, e estão para me responder. Não sei quanto tempo vai até me redirecionarem ao Banco Rendimento, então vou tentar entrar em contato diretamente com eles. Sabe me dizer que tipo de invoice eles requerem? Pode ser um do https://freeinvoicebuilder.com/, ou eles exigem uma nota fiscal ou algo do tipo?

**Reply by igorbp** [reply]
Eu enviava uma invoice que eu fiz mesmo. Tinha as informações da empresa e do cliente. Acho que essa aí serve. 

Eu usava a mesma invoice que eu enviava para o cliente.

**Comment by igorbp** [comment]
@GiovanniZambiasi pode ser essa invoice ai de boa eu acho. Eu uso uma num padrão também que o meu cliente me mandou.

**Comment by andreksu** [comment]
> Olá!
> 
> Utilizo o Payoneer para receber pagamentos em dólar do meu cliente (do Reino Unido). Tenho uma conta jurídica no Banco Inter, e vinha conseguindo receber no primeiro semestre de 2019. Depois de passar alguns meses sem usar o serviço, tentei fazer um saque para minha conta, como de costume. No entanto, após 5 dias úteis recebo um email dizendo que o requerimento falhou. Estou tentando resolver o problema, mas temo que não seja possível. Se for o caso, preciso de recomendações de bancos para abertura de conta PJ. Acredito que esses bancos digitais tenham maiores dificuldades com câmbio, visto que o próprio banco Inter para fazer remessas internacionais te dá uma dor de cabeça atrás da outra. Alguém teria recomendação de alguma conta PJ que funcione com Payoneer? Se não, alguma conta PJ que me permita receber em dólar sem cobrar nenhuma taxa muito alta?
> 
> Desde já agradeço!
> Giovanni

Pois é, eu também presto serviço para uma empresa americana e recebia pelo Banco Inter, porém eles resolveram mudar o canal bancário do câmbio e depois disso nunca mais consegui receber meu dinheiro, já que a empresa não conseguia efetuar a transferência de forma alguma. Tentei entrar em contato com o Banco Inter e eles não queriam me ajudar, perguntei também se tinham algum funcionário/atendente que soubesse falar inglês para entrar em contato com a empresa e mesmo assim se recusaram a me ajudar, enviando apenas mensagens prontas. Eu a partir daí nunca mais usei o Inter. Atualmente estou no Banco do Brasil e está muito bom... estou recebendo sem problemas e com diferença pequena no dólar.

**Comment by andreksu** [comment]
Para vocês que usam o Payoneer, quanto vcs pagam de tarifas? E qual a diferença das cotações?

**Comment by GiovanniZambiasi** [comment]
> Pois é, eu também presto serviço para uma empresa americana e recebia pelo Banco Inter, porém eles resolveram mudar o canal bancário do câmbio e depois disso nunca mais consegui receber meu dinheiro, já que a empresa não conseguia efetuar a transferência de forma alguma. Tentei entrar em contato com o Banco Inter e eles não queriam me ajudar, perguntei também se tinham algum funcionário/atendente que soubesse falar inglês para entrar em contato com a empresa e mesmo assim se recusaram a me ajudar, enviando apenas mensagens prontas. Eu a partir daí nunca mais usei o Inter. Atualmente estou no Banco do Brasil e está muito bom... estou recebendo sem problemas e com diferença pequena no dólar.

O engraçado é que o saque falhou mesmo para a minha conta pessoal da caixa (que sempre funcionou). Está usando o BdoB com o Payoneer? Se não, quais são as tarifas? Valeu!

**Comment by samiraguiar** [comment]
@andreksu recebo pela Payoneer e pago 2% de spread. Não tem outras tarifas porque recebo em Euro, mas pra quem recebe em dólar americano pode ser cobrado até 1% a mais. A cotação que eles utilizam é a [mid-market](https://www.xe.com/faq/rates_buyorsellrates.php).
O saque pra conta bancária é gratuito e via TED, então costuma cair alguns minutos depois.

**Comment by maribmota75** [comment]
olá pessoal, tudo bem? começarei a usar o payoneer (me cadastrando agora) e vi que o esquema vai ser exatamente o mesmo que vocês descreveram. tenho uma dúvida: você consegue ter a conta individual (em vez de empresarial) no Payoneer e enviar pra PJ do Inter? Não precisa ter conta empresa no payoneer? 

Obrigada e desculpa se não estiver tão clara a pergunta.

**Comment by GiovanniZambiasi** [comment]
> olá pessoal, tudo bem? começarei a usar o payoneer (me cadastrando agora) e vi que o esquema vai ser exatamente o mesmo que vocês descreveram. tenho uma dúvida: você consegue ter a conta individual (em vez de empresarial) no Payoneer e enviar pra PJ do Inter? Não precisa ter conta empresa no payoneer?
> 
> Obrigada e desculpa se não estiver tão clara a pergunta.

Olá @maribmota75!

Tu até pode receber como PF na conta Payoneer, e transferir para uma conta PJ em reais. No entanto, eu não sei que tipo de implicações fiscais isso teria. O ideal é que a empresa receba o dinheiro da empresa, e tu só receba o dinheiro do pró labore. Eu criei uma conta jurídica no Payoneer, e não tem muita burocracia (foi praticamente igual à conta PF), e funciona da mesma maneira com as mesmas taxas.

Agora, se quiser pagar um pouco menos de taxa, pode usar o serviço do RemessaOnline for buisness. É um canal de câmbio brasileiro que cobra somente 1.3%, taxa menor que os 2% do Payoneer. Em contrapartida, é um pouco mais burocrático (precisa gerar invoice, apresentar documentação legal, etc).

**Comment by maribmota75** [comment]
obrigada pela resposta, @GiovanniZambiasi 

o que enrola um pouco é a questão fiscal mesmo. vou verificar com a contabilidade.

é que acabei de ser contratada por uma startup americana e eles fazem o pagamento por paypal ou payoneer. vi que compensa pelo payoneer, mas não confirmei ainda com eles se faz diferença/precisaríamos fazer alguma alteração contratual pra receber como PJ direto no payoneer. vou checar, mas você já me esclareceu essa questão do payoneer!


Mutio obrigada! :)

**Comment by GiovanniZambiasi** [comment]
> > Pois é, eu também presto serviço para uma empresa americana e recebia pelo Banco Inter, porém eles resolveram mudar o canal bancário do câmbio e depois disso nunca mais consegui receber meu dinheiro, já que a empresa não conseguia efetuar a transferência de forma alguma. Tentei entrar em contato com o Banco Inter e eles não queriam me ajudar, perguntei também se tinham algum funcionário/atendente que soubesse falar inglês para entrar em contato com a empresa e mesmo assim se recusaram a me ajudar, enviando apenas mensagens prontas. Eu a partir daí nunca mais usei o Inter. Atualmente estou no Banco do Brasil e está muito bom... estou recebendo sem problemas e com diferença pequena no dólar.
> 
> O engraçado é que o saque falhou mesmo para a minha conta pessoal da caixa (que sempre funcionou). Está usando o BdoB com o Payoneer? Se não, quais são as tarifas? Valeu!

**[SOLUÇÃO]** Para quem encontrar o mesmo problema:

Depois de inúmeros contatos com o SAC do Payoneer, descobri que o problema era uma funding source (Viagogo). Em algum momento desse ano, eu conectei o Viagogo ao meu perfil do Payoneer. Porém, o Viagogo não permite transferências para o Brasil (mesmo que o dinheiro não tenha vindo deles). O SAC também me informou que funding sources não podem ser removidas, e me recomendou que criasse uma nova conta do Payoneer, transferisse o dinheiro da conta antiga para a nova e, a partir da nova conta, tentar o saque. Foi o que eu fiz, e deu certo.

Impressionante que precisei de 5 contatos através chat, mais quatro tentarivas por email para que a equipe Payoneer diagnosticasse corretamente o meu problema.

**Comment by samiraguiar** [comment]
@maribmota75 acho que por enquanto vc consegue fazer isso pela Payoneer, mas não sei por quanto tempo eles vão permitir

Recebi o seguinte e-mail esses dias:

> Due to Brazilian banking regulations, the type of bank account to which you withdraw funds from your Payoneer account must align with your Payoneer account type. If your account types are not aligned by December 15, 2019, you will not be able to withdraw funds.
> 
> Currently, your Payoneer account type is Company, meaning you will only be able to withdraw funds to a Business bank account.
> 
> If you already have a Business bank account connected to your Payoneer account, please withdraw your funds to this account from December 15, 2019. If you do not have a Business bank account, please add one to your Payoneer account. 

Eu recebi isso somente no e-mail da conta empresarial que tenho com eles, e não no e-mail da conta individual, então talvez isso não se aplique no cenário conta individual pra conta PJ, mas pode ser que no futuro eles bloqueiem.

**Comment by clmadrazo** [comment]
Já não é possível fazer transferências para contas PJs desde contas individuais de payoneer. É necessário ter uma conta de empresa no payoneer. 

Em 2 de dez de 2019, à(s) 15:06, maribmota75 <notifications@github.com> escreveu:

**Comment by felipebormann** [comment]
Galera, o código do banco inter no payoneer é mesmo 7277? E o branch que vocês colocaram foi 1 ou o último dígito da conta PJ de vocês?

**Comment by brunobarretosa** [comment]
> @andreksu recebo pela Payoneer e pago 2% de spread. Não tem outras tarifas porque recebo em Euro, mas pra quem recebe em dólar americano pode ser cobrado até 1% a mais. A cotação que eles utilizam é a [mid-market](https://www.xe.com/faq/rates_buyorsellrates.php).
> 
> O saque pra conta bancária é gratuito e via TED, então costuma cair alguns minutos depois.

O Remessa For Business(solução PJ do Remessa Online) cobra 1,4% default. Mas há um ano eu pago 1,105% em média por relacionamento!
E recebo em 24h após meu empregador enviar a remessa. Só eu emito a invoice e falo pro meu gerente no whatsapp pra ele colocar a TED pro dia seguinte! 🙌🏻

**Comment by enyamada** [comment]
@brunobarretosa Como vc conseguiu essa taxa mais baixa com Remessa Online? Faz mais de uma no que estou com eles e nunca me comentaram nada a respeito. obg

**Comment by marcosvieiraftw** [comment]
@enyamada Mês passado também recebi minha primeira remessa pelo Remessa Online. Criei a conta PJ com eles e pra primeira remessa consegui desconto. Eles dão cupom de 20% de desconto na taxa se você receber remessas mensais com eles (não sei com qual frequência se enquadra no mensal). Não sei se eles divulgam isso mas eu tive que correr atrás na negociação.

Depois de analisar muito as opções fiquei entre Inter e Remessa Online. Tinha escolhido o inter por ter a conta PJ lá e evitar mais 1 passo que seria a transferência, mas eles levaram quase 1 mês pra criar meu canal bancário e no meio do caminho chutei o balde e fiz o cadastro no Remessa Online em 24 horas. 

Sobre o Payoneer caso vocês estejam lendo hoje esse comentário agora **restritamente** Business envia para Conta Bancária PJ e individual envia para Conta Bancária PF. Não pode mais misturar.. tive que fechar minha conta individual pra conseguir abrir a business porque sou empresário individual e minha empresa leva meu nome e eles não aceitam 2 contas - mesmo que diferentes - com o mesmo nome.

_Uma dúvida_: **Alguém aqui tem conta business do Payoneer recebendo em Dólar Canadense? Se sim, como é o spread? é fixo nos 2% ou de acordo com o montante tem variação?**

A empresa que estou prestando serviço prefere trabalhar com deposito local a ter que enviar remessa e pra isso a conta bancária no canadá que o Payoneer dispõem ficaria interessante.

**Comment by pedrohyvo** [comment]
Quem souber e puder contribuir nessa issue #112 aqui também, seria muito bom :D

**Comment by ACM459** [comment]
Boa noite, pessoal, Me permitem uma questão ligeiramente off-topic? Eu gostaria de saber se eu receber o pagamento pela payonner, eu sou obrigado a sacar tudo para uma conta em reais, ou posso deixar parte do valor lá, tipo esperando uma cotação mais favorável do dólar pra sacar. Agradeço desde já!

**Comment by pedrohyvo** [comment]
> Boa noite, pessoal, Me permitem uma questão ligeiramente off-topic? Eu gostaria de saber se eu receber o pagamento pela payonner, eu sou obrigado a sacar tudo para uma conta em reais, ou posso deixar parte do valor lá, tipo esperando uma cotação mais favorável do dólar pra sacar. Agradeço desde já!

@ACM459 até onde eu sei sim, você só vai precisar pagar todos os impostos referente a sua empresa quando esse dinheiro for transferido para a sua conta PJ aqui no Brasil.

**Comment by lima-emanuel** [comment]
Eu tenho uma dúvida com relação a isso. Então se eu mantiver uma conta em dólares no exterior eu posso nunca declarar nem nunca pagar impostos se eu nunca trouxer esse dinheiro para cá? Ou pagar menos impostos se eu só trouxer uma parte? Parece ilegal, então talvez não né rsrs

**Comment by heliomeiralins** [comment]
totalmente ilegal. ahhahaha

Em seg., 25 de jan. de 2021 às 09:35, Emanuel Lima <notifications@github.com>
escreveu:

**Comment by lucasven** [comment]
@emanuellima1 consulte um bom contador pra sua dúvida. Pesquise sobre elisão fiscal. Existem manobras pra pagar menos impostos que não são crime.

**Comment by levilugato** [comment]
Fala pessoal, estou a um tempo tentando entender se eu conseguiria trocar o remessa on line pela payoneer

trabalho pra uma empresa de UK e recebo como PJ no BR utilizando o remessa on line for business mas queria uma corretora que me permitisse fracionar e manter uma parte da grana la

pra que eu possa utilizar o payoneer pra essa finalidade a empresa que presto serviço precisa ter algum tipo de vínculo com a payoneer ou simplesmente posso abrir a conta la q eles me dão uma conta bancária de la e eu posso mandar pra minha conta PJ daqui do BR 

Vlw

**Comment by pedrohyvo** [comment]
@levilugato caso a sua empresa tenha o Payoneer Enterprise você deve pagar menos taxas(cerca de uns 3 dólares por transação) na hora de receber, como acredito que somente você esteja com essa situação, você vai pagar uma taxa de 5% do valor total sempre que receber o salário na sua conta da Payoneer.

**Comment by igorlvicente** [comment]
Veja se vale à pena esses 5%. Se eu calculei certo, o Remessa Online não passa dos 2% de spread. Junte isso a uma das contas internacionais que têm hoje em dia (Banco BS2 e Banco C6) e talvez a dupla conversão nem chegue a ser 5%.

**Comment by levilugato** [comment]
Obrigado @pedrohyvo e @igorlvicente , então, o que pega pra mim no Remessa on line fb é o fato de não deixarem fracionar (tenho sempre que trocar o salário todo) a remessa ou até mesmo transferir pra uma outra conta que não seja a minha conta PJ no BR e como pretendo passar um tempo na europa em breve creio que uma abordagem do tipo da payoneer seria mais interessante pra mim

Grato pelas informações

**Comment by samiraguiar** [comment]
@levilugato a Payoneer não cobra taxa pra recebimento em libra ou euro (diferentemente de dólar), só cobra um spread de 2% quando você transfere para o Brasil. Não tem IOF nem outras cobranças, mas dá uma olhada [no site](https://www.payoneer.com/about/fees/) pra saber mais.

Ela te dá sim uma conta pra recebimento em libra, e o cliente pode fazer uma transferência comum. Tem três ressalvas:
- são aceitas transferências somente a partir de contas business (ou seja, contas "individual" ou "PF" como chamamos aqui não podem transferir pra você)
- o primeiro pagamento vai ficar retido e vão solicitar mais informações. Eles vão pedir contrato de prestação de serviço, invoice, trocas de e-mail e outros comprovantes pra verificarem que é mesmo uma relação de trabalho e não lavagem de dinheiro, operação libra-cabo, etc. Essa análise costuma levar até uns 5 dias úteis, e depois não pedem mais para receber do mesmo remetente
- pra trazer os valores para o Brasil, a Payoneer usa dois intermediários - a USend e o Banco Rendimento. Em algum ponto pode ser que eles também solicitem documentos pra liberar seu dinheiro aqui. Comigo levou três anos para solicitarem, com outros levou menos, mas também é tranquilo - só mandar tudo por e-mail e em poucos dias liberam. Costuma ser coisas simples - invoices, contrato de trabalho, balanço patrimonial, etc. No meu caso pediram uma vez só também e nunca mais.

**Comment by levilugato** [comment]
@samiraguiar muitíssimo obrigado pelas informações foram de grande valia, ja até fiz o cadastro lá
sobre as ressalvas que você citou

1.  eu só posso receber de contas business mas poderia transferir da minha conta Jurídica pra uma física por ex ?
2. Estas análises acredito que são comuns mesmo
3. Você vê algum problema jurídico quanto à quantia trazida pro Br (digo mais pela diferença caso eu opte aleatóriamente por deixar uma parte da grana lá) ?

Abraço

**Comment by samiraguiar** [comment]
@levilugato opa, sem problemas

> 1. eu só posso receber de contas business mas poderia transferir da minha conta Jurídica pra uma física por ex ?

Pra conta bancária não, também tem que ser conta jurídica. Você pode abrir uma conta como PF na Payoneer e pagar a PF a partir da PJ, mas acho que isso seria meio que uma gambiarra.

> 3. Você vê algum problema jurídico quanto à quantia trazida pro Br (digo mais pela diferença caso eu opte aleatóriamente por deixar uma parte da grana lá)?

Não vejo não. Dá uma lida [nessa discussão](https://github.com/DyegoCosta/trabalhando-remoto/discussions/122) aqui.

**Comment by hsmagnago** [comment]
Venho usando Payoneer a alguns anos. Comecei recebendo direto na PF e declarando como PF. A alguns meses criei uma PJ e criei uma nova conta da Payoneer para a PJ, para reduzir os impostos. Fiz diversos saques sem problema algums já com a PJ. A 2 dias recebi uma ligação da USEND falando que iriam me enviar um e-mail e que precisavam de mais documentos para liberar a transferência.

Eu nunca tinha ouvido falar da USEND/Banco Rendimento. Achei bem estranho o pedido deles e achei até que era golpe. Mas já fazem 3 dias e realmente o dinheiro ainda não caiu na minha conta. @samiraguiar onde tu encontrou essa informação que o Payonner use eles como intermediários? Tava no chat aqui do Payonner e nem eles souberam me falar isso.

**Comment by pedrohyvo** [comment]
> Venho usando Payoneer a alguns anos. Comecei recebendo direto na PF e declarando como PF. A alguns meses criei uma PJ e criei uma nova conta da Payoneer para a PJ, para reduzir os impostos. Fiz diversos saques sem problema algums já com a PJ. A 2 dias recebi uma ligação da USEND falando que iriam me enviar um e-mail e que precisavam de mais documentos para liberar a transferência.
> 
> Eu nunca tinha ouvido falar da USEND/Banco Rendimento. Achei bem estranho o pedido deles e achei até que era golpe. Mas já fazem 3 dias e realmente o dinheiro ainda não caiu na minha conta. @samiraguiar onde tu encontrou essa informação que o Payonner use eles como intermediários? Tava no chat aqui do Payonner e nem eles souberam me falar isso.

@hsmagnago pode ficar tranquilo cara, eles vão te pedir uns documentos para comprovar que o dinheiro vem de forma lícita. Então de tempos em tempos eles fazem auditoria e pedem essa documentação para autorizar o envio da sua remessa para sua conta aqui no Brasil.

**Comment by samiraguiar** [comment]
@hsmagnago eles entraram em contato comigo por e-mail porque não conseguiram por telefone, e essas duas empresas estavam envolvidas (também tinha a Pontual Money Transfer, mas eles são os donos da USend). Se não me engano, quando transfiro para minha conta da Payoneer o dinheiro vem do Banco Rendimento, então imagino que a USend faça o câmbio e o Rendimento a transferência.

Mas como o Pedro disse é bem tranquilo, só enviar os documentos que eles pedirem e eles liberam depois de poucos dias.

**Comment by thalissonrosa** [comment]
Eu sinceramente não consigo mais recomendar o uso do Payoneer exatamente por conta do Banco Rendimento. Já faz mais de uma semana que eles estão com o meu dinheiro, já enviei os documentos e está uma enrolação enorme. Tirando a demora deles responderem qualquer coisa (e ignorar perguntas) a ultima novidade é que eles não querem aceitar documento assinado com certificado digital, o que é completamente ridículo.
Agora eles querem que o documento seja impresso, assinado, scaneado e enviado por email. Só na cabeça deles que isso é mais seguro que o uso de certificado digital.

**Comment by maxmaccari** [comment]
Eu também não recomendo a Payoneer. Eles simplesmente cancelaram um pagamento meu sem justificativa, e já faz uns 5 dias, e não reembolsaram a empresa que me pagou. O suporte deles é inexistente, pois abri chamado questionando, e não obtive uma resposta até hoje. A empresa é grande, mas o serviço não me passa confiança nenhuma.

**Comment by levilugato** [comment]
Pessoal, vocês indicam alguma alternativa à Payoneer que me de a mesma possibilidade ? (deixar uma parte do dinheiro la e converter facilmente pra outra moeda) ?

**Comment by lmcarreiro** [comment]
> Eu sinceramente não consigo mais recomendar o uso do Payoneer exatamente por conta do Banco Rendimento. Já faz mais de uma semana que eles estão com o meu dinheiro, já enviei os documentos e está uma enrolação enorme. Tirando a demora deles responderem qualquer coisa (e ignorar perguntas) a ultima novidade é que eles não querem aceitar documento assinado com certificado digital, o que é completamente ridículo.
> Agora eles querem que o documento seja impresso, assinado, scaneado e enviado por email. Só na cabeça deles que isso é mais seguro que o uso de certificado digital.

Está difícil essa enrolação do Banco Rendimento... Mas aqui, pelo menos no início em que pediram uma ficha cadastral, eu assinei ela no Adobe PDF sem certificado, aquela assinatura que vc usa o mouse pra simular uma caneta, sabe? Ridículo que aceitaram até isso...

**Comment by td-design** [comment]
Fala galera. Sou designer gráfico freelancer e presto serviços para várias empresas de vários países diferentes. E venho utilizando o Payoneer + Banco Inter PJ para receber os pagamentos, assim como muitos de vocês aqui. Normalmente para declarar o dinheiro que entra, eu emito um NF mensal com os dados da empresa no exterior com os valores convertidos em real. Porém fico muito confuso de se essa é a maneira mais correta de se fazer isso, como vocês fazem?

**Comment by lucasven** [comment]
isso, esse é o jeito de se fazer isso. Você não tem contador? sugiro contratar um contador de verdade, e não essas soluções de contabilidade online

**Comment by pedrohyvo** [comment]
@td-design se tiver interesse posso te passar um contato da minha contadora que ela atende o Brasil e ela vai tirar todas as suas dúvidas.

**Comment by td-design** [comment]
> isso, esse é o jeito de se fazer isso. Você não tem contador? sugiro contratar um contador de verdade, e não essas soluções de contabilidade online

Sim eu tenho, mas infelizmente a maioria dos contadores ainda não são especializados nesse tipo de negócio. Uma dúvida que tenho por exemplo é se caso eu mantiver o dinheiro sem converte-lo na minha conta da Payoneer, se eu precisaria declarar isso de alguma forma como PJ. Pois não consigo emitir NF em outras moedas.

Mas por outro lado, o Payoneer disponibiliza um cartão de credito, que permite usar os saldos em moedas diferentes sem precisar necessariamente converter nada para Real, mas usar esse dinheiro sem declarar soa muito ilegal. kkk

**Comment by lucasven** [comment]
Se manter o dinheiro na conta da payoneer não precisa pagar imposto realmente, tem que trazer pra ca ou colocar em reais pra ser declarado.

Quanto ao cartão de crédito da payoneer, é ilegal sim gastar dinheiro no BR que voce não declarou. Se o IR te pega fazendo isso tu tem que pagar retroativo o imposto. Mas a chance disso acontecer é minima. Voce teria que cair numa malha fina por ter X de compra no teu CPF e só ter declarado X/2.

Eu conheço gente que faz compra internacional ou viagem com esse dinheiro, isso não é ilegal. Porque seu pais de residência é aqui, você é obrigado a declarar o que está aqui e gasta aqui.

Se quiser manter em outros assuntos sugiro abrir uma nova thread

**Comment by viniciusdeliz** [comment]
@thalissonrosa @maxmaccari recomendam alguma alternativa ao Payoneer ou continuaram utilizando-o?

**Reply by pedrohyvo** [reply]
@viniciusdeliz eu estou usando Techfx, pago apenas 0.5% de taxa e o spread é o dolar comercial. Usa meu convite que a sua primeira transferência é de apenas 0.1% - https://app.techfx.com.br/referral/0tiachqg98y9

**Comment by heliomeiralins** [comment]
> Se manter o dinheiro na conta da payoneer não precisa pagar imposto realmente, tem que trazer pra ca ou colocar em reais pra ser declarado.

Isso não procede. O fato gerador de de NF fiscal em regime de competência é a geração de uma invoice. No regime de caixa é receber o dinheiro; seja numa conta BR ou gringa. Não existe isso de deixar o dinheiro fora não paga imposto. vc precisa declarar a NF com a cotação do dólar no dia. Tem muita gente que sempre declara o valor que entra na conta e acaba não entendo essa lógica.

Se vc deixar em dólar e tiver lucro cambial esse valor também precisa ser declarado. ( pelo menos no lucro presumido, no simples eu não sei como funciona )

**Comment by MagBluu** [comment]
Criei uma conta PJ no inter e no Payoneer, mas estou tendo problemas para enviar um documento que comprove minha titularidade da conta, já que a conta PJ brasileira vem com os números iniciais do seu CNPJ no nome do dono da conta, e o Payoneer não reconhece o nome como sendo igual o da conta. Não sei o que fazer, visto que esses números iniciais do CNPJ são parte da regulamentação da lei brasileira, e preciso dessa conta confirmada no Payoneer para receber meus pagamentos, alguém sabe como resolver?

**Comment by pedrohyvo** [comment]
Pessoal, já usei bastante Payoneer, mas agora estou usando Techfx, quem quiser se cadastrar usa o meu link que a primeira transação é 0.1%, a taxa deles é apenas 0.5% - https://app.techfx.com.br/referral/0tiachqg98y9
